CMakeLists.txt

```diff
@@ -0,0 +1,106 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(imgui)
+set(IMGUI_VERSION "1.62")
```

Comment:
Since CMake 3.1 is a requirement, you can use `project(... VERSION ...)` instead, will do the same.